extensions/markdown-language-features/src/util/links.ts

```diff
@@ -22,13 +22,29 @@ const knownSchemes = [
 ];
 
 export function getUriForLinkWithKnownExternalScheme(link: string): vscode.Uri | undefined {
-	if (knownSchemes.some(knownScheme => isOfScheme(knownScheme, link))) {
+	const cleanLink = stripAngleBrackets(link);
```

Comment:
Let's move the angle bracket stripping logic to the caller instead ( `parseLink`)  since angle brackets are a detail of the document link syntax

Reply:
I'm looking to move that over to `parseLink` instead 👍

Reply:
I moved the function and the call of it to `extensions\markdown-language-features\src\features\documentLinkProvider.ts`.  If that is not what you intended, please let me know